geles-frontend/src/EditOrder.svelte

```diff
@@ -0,0 +1,517 @@
+    switch (status) {
+      case "UNPAID":
+        return "Neapmokėta";
+      case "PAID":
+        return "Apmokėtas";
+      case "CONFIRMED":
+        return "Patvirtintas";
+      case "DELIVERED":
+        return "Pristatytas";
+      case "CANCELED":
+        return "Atšauktas";
+      default:
+        return "???????";
+    }
+  }
+
+  // Run code on component mount (once)
+  onMount(async () => {
+    await axios.get(`/flowers/`).then(resp => {
+      flowers = resp.data;
+    });
+    await getOrderData();
+    await verifyLogin();
+    if (isAdmin) await loadUser();
+  });
+</script>
+
+<h2>Užsakymas</h2>
+<div class="row">
+  <div class="column">
+    <div class="flowers-table">
+      <table>
+        <tr>
+          <th colspan="2">Gėlė</th>
+          <th>Kiekis</th>
+          <th>Vnt. kaina</th>
+          <th>Suma</th>
+          {#if order.orderStatus == "UNPAID"}
+            <th>Veiksmai</th>
+          {/if}
+        </tr>
+        {#each editDto.orderFlowers as flower (flower.id)}
+          <tr>
+            <div class="imagecontainer">
+              {#if flowerFromId(flower.flowerId).photo != null}
+                <img
+                  class="flower-list-item-photo"
+                  src={`${server_url}/static/flowers/${
+                    flowerFromId(flower.flowerId).photo
+                  }`}
+                  alt={flowerFromId(flower.flowerId).name}
+                  width="80"
+                  height="80"
+                />
+              {/if}
+            </div>
+            <td>{flowerFromId(flower.flowerId).name}</td>
+            <td>
+              {#if order.orderStatus == "UNPAID"}
+                <input
+                  type="number"
+                  bind:value={flower.quantity}
+                  min="1"
+                  max="100"
+                  size="5"
+                  on:input={e => {
+                    recalcSum();
+                  }}
+                />
+              {:else}
+                {flower.quantity}
+              {/if}
+            </td>
+            <td class="number">{flowerFromId(flower.flowerId).price} €</td>
+            <td class="number">{rowSum(flower).toFixed(2)} €</td>
+            {#if order.orderStatus == "UNPAID"}
+              <td>
+                <button on:click={() => handleDelete(flower)}>Pašalinti</button>
+              </td>
+            {/if}
+          </tr>
+        {/each}
+      </table>
+      <div class="row buttonsrow">
+        <button
+          class="button savebutton"
+          disabled={disableButton(order.orderStatus, OrderEditButton.SAVE)}
+          on:click={() => handleUpdate()}
+        >
+          Išsaugoti pakeitimus
+        </button>
+        <button
+          class="button paybutton"
+          disabled={disableButton(order.orderStatus, OrderEditButton.PAY)}
+          on:click={() => handlePay()}
+        >
+          Apmokėti
+        </button>
+
+        <button
+          class="button cancelbutton"
```

Comment:
If buttons already have `button` class, seems unnecessary to write `button` in front of modifier classes